server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java

```diff
@@ -0,0 +1,113 @@
+package com.objectcomputing.checkins.services.pulseresponse;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+@Entity
+@Table(name="pulse_response")
+public class PulseResponse {
+    
+    public PulseResponse() {}
+
+    public PulseResponse(LocalDate submissionDate, LocalDate updatedDate, UUID teamMemberId, String questionResponse, String internalFeelings, String externalFeelings) {
+        this.submissionDate = submissionDate;
+        this.updatedDate = updatedDate;
+        this.teamMemberId = teamMemberId;
+        this.questionResponse = questionResponse;
```

Comment:
This is actually not quite right. I'd remove questionResponse for now. There will be multiple questions...not just one, and as a result the answers will need to be stored in a separate table.